scheduler/src/cook/mesos/api.clj

```diff
@@ -872,7 +872,7 @@
                         jobs (get params :jobs)
                         groups (get params :groups)
                         user (get-in ctx [:request :authorization/user])
-                        override-group-immutability? (boolean (get params override-group-str))]
+                        override-group-immutability? (boolean (get params :override-group-immutability))]
```

Comment:
You are right that this makes a difference - we are fixing something that was broken (string key is never present here).

But yes, I will add unit tests for this.